core/dbt/config/project.py

```diff
@@ -45,7 +45,7 @@
 This version of dbt is not supported with the '{package}' package.
   Installed version of dbt: {installed}
   Required version of dbt for '{package}': {version_spec}
-Check the requirements for the '{package}' package, or run dbt again with \
+Check for an updated version of the '{package}' package, or run dbt again with \
```

Comment:
Would there be any scenario where we would need an older version of a package and not the newest? Honestly not sure so just asking

Reply:
Interesting! I guess if you started the 1.0 upgrade process, changed your dbt-utils to 0.8.0, then abandoned ship and went back to 0.21, then you'd run into the same error message and should technically go to an _older_ version...

This assumes that you merged your changes into main before you were confident in the entire migration which sounds ill-advised, but is possible.

Reply:
@leahwicz i about it more, I'd also be on board with the phrasing "a different version" if we really wanted to protect against that edge case

Reply:
Thanks @joellabes! I wasn't sure how likely it was to happen. I just was worried about giving too specific of instructions and have them be incorrect for an edge case